lib/Froxlor/Api/Commands/Mysqls.php

```diff
@@ -79,7 +81,12 @@ public function add()
 		);
 		// create database, user, set permissions, etc.pp.
 		$dbm = new \Froxlor\Database\DbManager($this->logger());
-		$username = $dbm->createDatabase($newdb_params['loginname'], $password, $newdb_params['mysql_lastaccountnumber']);
+
+		if(strtoupper(Settings::Get('customer.mysqlprefix')) == 'DBNAME' && !empty($databasename)) {
+			$username = $dbm->createDatabase($newdb_params['loginname'].'_'.$databasename, $password);
```

Comment:
At this point I have to replace the hard-coded underline.

Reply:
why? it's an allowed character and makes total sense if the name is prefixed with the customer's loginname